codalib/bagatom.py

```diff
@@ -42,7 +42,8 @@ def wrapAtom(xml, id, title, author=None, updated=None, author_uri=None, alt=Non
             entryTag,
             ATOM + "link",
             rel='alternate',
-            href=alt)
+            href=alt,
+            type="text/html")
```

Comment:
Now that I am seeing the `type="text/html"` here I am wondering if it is kind of dangerous to hard code, since I don't think the `alt` URI would have to be `text/html`.